e2e/metrics/bundledeployment_test.go

```diff
@@ -24,26 +23,31 @@ var _ = Describe("BundleDeployment Metrics", Label("bundledeployment"), func() {
 		// objName is going to be "randomized" instead of using a dedicated and
 		// random namespace, like it is the case for the other tests.
 		objName string
+
+		r = rand.New(rand.NewSource(GinkgoRandomSeed()))
```

Comment:
GinkgoRandomSeed() is unique, even among parallelized tests, which means when running those tests in parallel, they will get the same seed, generate the same randomization source and create the same random names, thereby breaking parallelism of tests.

Reply:
Do you have any source for that?
I quickly tested it and it seems to work:

```
package metrics_test

import (
	"fmt"
	"math/rand"

	. "github.com/onsi/ginkgo/v2"
)

var _ = FDescribe("Bundle Metrics", Label("bundle"), func() {
	var r = rand.New(rand.NewSource(GinkgoRandomSeed()))

	It("a", func() {
		fmt.Printf("a random: %d\n", r.Intn(100))
		fmt.Printf("a random: %d\n", r.Intn(100))
	})
	It("b", func() {
		fmt.Printf("b random: %d\n", r.Intn(100))
		fmt.Printf("b random: %d\n", r.Intn(100))
	})
	It("c", func() {
		fmt.Printf("c random: %d\n", r.Intn(100))
		fmt.Printf("c random: %d\n", r.Intn(100))
	})
})
```

However, what you cannot do, is re-create the source from the same seed in `BeforeEach`.